pickfirst.go

```diff
@@ -161,7 +163,7 @@ type picker struct {
 	err    error
 }
 
-func (p *picker) Pick(info balancer.PickInfo) (balancer.PickResult, error) {
+func (p *picker) Pick(_ balancer.PickInfo) (balancer.PickResult, error) {
```

Comment:
Nit: delete parameter name entirely instead of using "_".  And below.

Reply:
Done.